org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/META-INF/MANIFEST.MF

```diff
@@ -4,12 +4,18 @@ Bundle-Name: jdt.ls.extension Core Extension
 Bundle-SymbolicName: jdt.ls.extension.core;singleton:=true
 Bundle-Version: 0.0.1.qualifier
 Bundle-Activator: org.eclipse.che.jdt.ls.extension.core.internal.ExtensionActivator
-Import-Package: org.eclipse.jdt.core,
+Import-Package: com.google.gson,
+ org.eclipse.che.jdt.ls.extension.api,
+ org.eclipse.che.jdt.ls.extension.api.dto,
  org.eclipse.core.resources,
+ org.eclipse.jdt.core,
  org.eclipse.jdt.launching,
+ org.eclipse.lsp4j,
+ org.eclipse.lsp4j.jsonrpc.json.adapters,
  org.osgi.framework;version="1.3.0"
 Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.jdt.core,
- org.eclipse.jdt.ls.core
+ org.eclipse.jdt.ls.core,
+ org.eclipse.jdt.core.manipulation
```

Comment:
would it help to use maven-bundle-plugin and let BND generate the correct import packages.